lib/pthreads/threadpool.nit

```diff
@@ -23,21 +23,22 @@ class ThreadPool
 	private var queue = new ConcurrentList[Task]
 	private var mutex = new Mutex
 	private var cond = new NativePthreadCond
+	private var threads = new Array[PoolThread]
 
-	# Number of threads used
-	var nb_threads: Int is noinit
+	# Number of threads used, can only grow after the first call to `execute`
```

Comment:
From the public API I would say no, but from within the class one could check `threads.not_empty`.